datahub-web-react/src/app/search/sorting/SearchSortSelect.tsx

```diff
@@ -0,0 +1,51 @@
+import Icon, { CaretDownFilled } from '@ant-design/icons';
+import { Select } from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import { ReactComponent as SortIcon } from '../../../images/sort.svg';
+import { DEFAULT_SORT_OPTION, SORT_OPTIONS } from '../context/constants';
+import { useSearchContext } from '../context/SearchContext';
+
+const SelectWrapper = styled.span`
+    display: inline-flex;
+    align-items: center;
+    margin-right: 8px;
+
+    .ant-select-selection-item {
+        // !important is necessary because updating Select styles for antd is impossible
+        color: ${(props) => props.theme.styles['primary-color']} !important;
+        font-weight: 700;
+    }
+
+    svg {
+        color: ${(props) => props.theme.styles['primary-color']};
+    }
+`;
+
+const StyledIcon = styled(Icon)`
+    color: ${(props) => props.theme.styles['primary-color']};
+    font-size: 16px;
+    margin-right: -6px;
+`;
+
+export default function SearchSortSelect() {
+    const { selectedSortOption, setSelectedSortOption } = useSearchContext();
```

Comment:
Could be another opportunity for some custom hooks, non blocking feedback here of course!

Reply:
what would the benefit of using two different custom hooks here? at the end of the day I'm just trying to get these values from our context

Reply:
In this example, not much. But as contexts grow and become more complex, having less usage of the global context and more specific selection of the values becomes cleaner imo.